src/utils/conditionToDatalog.ts

```diff
@@ -186,6 +209,30 @@ const translator: Record<string, Translator> = {
     placeholder: "Enter any placeholder for the node",
     isVariable: true,
   },
+  "is referenced by block in page with title": {
```

Comment:
this is a mouthful, but the alternative was reworking the `not` condition to be able to handle multiple clauses